src/components/CodeBlockGroupIcon.vue

```diff
@@ -1,8 +1,8 @@
 <template>
   <a
     :href="link"
-    target="_blank"
-    rel="noopener nofollow"
+    :target="linkNewTab ? '_blank' : '_self'"
+    :rel="linkNewTab ? 'noopener nofollow' : ''"
```

Comment:
We need to use router-link for internal navigation. You can use a pattern match to see if the link is relative and either render an `<a>` or ` <router-link>` either way.

Reply:
For example: if link starts with `/` then use `<router-link :to="link">` otherwise use `<a :href="link" target="_blank" rel="noopener nofollow">`

Reply:
Updated, thanks for the suggestion.